tests/template/tmodulealias.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  disabled: true
+    disabled: true
```

Comment:
Disabled around 2011-05 in a commit that references #30 and #26
probably delete, 10 years already.

Reply:
Exterminate!